docs/proposals/Keadm Tool Enhancement.md

```diff
@@ -0,0 +1,35 @@
+# Keadm Tool Enhancement
+
+## Motivation
+
+Keadm(KubeEdge installation tool) now only supports configuring a subset of parameters during EdgeCore installation.
+
+We would like to support specifying parameters using `--set` or directly using an existing local configuration file to achieve full parameter configuration and meet the users' requirements. 
+
+## Goals
+
+- Use '--set' or local configuration file to set parameters
+
+## Design Details
+
+**The frame design drawing is as follows**：
+
+![](https://raw.githubusercontent.com/HT0403/Pictures/master/img/202404262037218.png)
+
+1. **Start** - The entry point of the program which takes in two inputs：
```

Comment:
This content seems to be related only to the `set`. Can you describe it in detail based on the current keadm capabilities and code?